From 4939f61054dfd34197f92f6730fb69f5791eeefa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Bernon?= <rbernon@codeweavers.com>
Date: Tue, 28 Sep 2021 10:49:53 +0200
Subject: [PATCH 1/4] configure.ac: Add -fasynchronous-unwind-tables to
 CROSSLDFLAGS.

When available. Otherwise the output_cfi calls in winebuild, in the
syscall thunks for instance, are just no-op.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 5074ee14833..95d9127d8fd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1906,6 +1906,7 @@ then
    DLLFLAGS="$DLLFLAGS -fasynchronous-unwind-tables"
    LDDLLFLAGS="$LDDLLFLAGS -fasynchronous-unwind-tables"
    UNIXDLLFLAGS="$UNIXDLLFLAGS -fasynchronous-unwind-tables"
+   CROSSLDFLAGS="$CROSSLDFLAGS -fasynchronous-unwind-tables"
 else
    if test "x$enable_win64" = "xyes"
    then
-- 
2.34.0

